topology/topology.cpp

```diff
@@ -55,6 +55,46 @@ LayerMetadata::LayerMetadata( AbstractLayerPTR layer )
 {
 }
 
+void
+LayerMetadata::slice( size_t start, size_t stop, size_t step, NodeCollectionPTR node_collection )
+{
+  // Get positions of current layer, sliced in start-stop.
```

Comment:
I think it might be an idea to also add a comment stating why you aren't stepping, similar to the comment in `SetStatus`, in case people have not read that comment.

Reply:
Done!